records_to_socrata.py

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env python
+import argparse
+import os
+import arrow
+import utils
+
+PGREST_ENDPOINT = os.getenv("PGREST_ENDPOINT")
+PGREST_TOKEN = os.getenv("PGREST_TOKEN")
+SOCRATA_RESOURCE_ID = os.getenv("SOCRATA_RESOURCE_ID")
+
+
+def format_filter_date(date_from_args):
+    return "1970-01-01" if not date_from_args else arrow.get(date_from_args).isoformat()
+
+
+def main(args):
+    filter_iso_date_str = format_filter_date(args.date)
+
+    client_postgrest = utils.postgrest.Postgrest(PGREST_ENDPOINT, token=PGREST_TOKEN)
+
+    data = client_postgrest.select(
+        "traffic_reports",
+        params={
+            "published_date": f"gte.{filter_iso_date_str}",
+        },
+        order_by="traffic_report_status_date_time",
+    )
+
+    logger.info(f"{len(data)} records to process")
+
+    if not data:
+        return
+
+    client_socrata = utils.socrata.get_client()
+    method = "replace" if not args.date else "upsert"
```

Comment:
im wondering if we ever would want to do a full replace, there are currently 71,000 records in the db

Reply:
the airflow dag is written to not do a replace once a month, but i am thinking that it should also be prevented here?

Reply:
ah I see that we have 354k records in Socrata, yeah we probably want to not do this unless we backfilled the missing data in postgres?

Reply:
Could just set the `filter_iso_date_str` to 1-1-1970 if the arg is not supplied?